src/clims/api/endpoints/user_files.py

```diff
@@ -27,7 +20,7 @@ def get(self, request):
 
     def post(self, request):
         content = request.DATA["content"]
-        file_name = request.DATA["fileName"]
+        # file_name = request.DATA["fileName"]
```

Comment:
Perhaps add a comment here on why it is commented out rather than removed?

Reply:
Should have done that :) But this will be revisited soon (TM).